graphql/types/Product.graphql

```diff
@@ -261,9 +261,29 @@ type Offer {
   discountHighlights: [Discount!]
   teasers: [Teaser!]
   """
-  List of gifts associated with the product
+  List of SKUs for gifts associated with the product
   """
   giftSkuIds: [String]
+  """
+  List of gifts associated with the product
+  """
+  gifts: [Gift]
+}
+
+type Gift {
+  nameComplete: String
+  productName: String
+  brand: String
+  linkText: String
+  description: String
+  productTitle: String
```

Comment:
What is this field? I think we should have only

```diff
- nameComplete
  productName
+ skuName
- productTitle
```

What do you think?

Reply:
I was thinking about using `productTitle` for the `gift-image` `<img>` tag.
But I guess those two you pointed out could be merged and I do agree that `skuName` is a much better name.